client/src/components/AddNewProfile.js

```diff
@@ -54,6 +54,11 @@ const AddNewProfile = () => {
     updateField(data.name, data.value);
   };
 
+  const nationalityHandler = (event, data) => {
+    const selectedNationality = data.value;
+    setProfileData({ ...profileData, nationality: selectedNationality });
+  };
+
```

Comment:
Do we need a separate handler function here? Why isn't the `handleDropdownAndDateChange` function that was used before up to the job?